src/EventStore.php

```diff
@@ -264,7 +238,7 @@ public function loadEventsByMetadataFrom(StreamName $streamName, array $metadata
      * @return CompositeIterator
      * @throws Exception\InvalidArgumentException
      */
-    public function replay(array $streamNames, DateTimeInterface $since = null, array $metadatas = null)
+    public function replay(array $streamNames, DateTimeInterface $since = null, array $metadatas = null): CompositeIterator
```

Comment:
Why `CompositeIterator`? Does it make sense from a consumer perspective? Also, the docblock is not needed anymore.